simphony/cuds/materials.py

```diff
@@ -0,0 +1,99 @@
+from simphony.cuds.material import Material
+
+
+class Materials(object):
+    """A class that contains materials.
+
+    Class manages the materials.
+
+    """
+    def __init__(self, uid=None, data=None, description=""):
```

Comment:
But what is the use of these attributes? The init method does not create these attributes to the ```Materials``` class.